httpbp/prometheus.go

```diff
@@ -107,6 +107,17 @@ var (
 	}, clientActiveRequestsLabels)
 )
 
+var (
+	panicRecoverLabels = []string{
+		methodLabel,
+	}
+
+	panicRecoverCounter = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "httpbp_server_panic_recover_total",
```

Comment:
Hmm, `bp` seems a bit uncessary in this naming. If this is an `http` server, it should just be under the `http` namespace. Also we tend to imply `server` for most metrics that are not explicitly `client`.

```suggestion
		Namespace: "http",
		Name: "panics_recover_total",
```